.github/workflows/topos:integration-tests.yml

```diff
@@ -67,18 +68,23 @@ jobs:
           cd infra &&
           npm install
 
+      - name: Setup environment
+        run: |
+          cd infra
+          cat .env | grep -v '^\s*$\|^\s*\#' | xargs -I {} echo "{}" >> $GITHUB_ENV
```

Comment:
This is… not great? Seems like a hack to me.

Reply:
Do you have a better solution?